integtest/spec/all_books_change_detection_spec.rb

```diff
@@ -45,15 +55,37 @@ def self.build_one_book_out_of_one_repo_twice(
       )
       build_twice(
         before_first_build: lambda do |src, config|
-          repo = src.repo_with_index 'repo', 'Some text.'
-          book = src.book 'Test'
-          book.source repo, 'index.asciidoc'
+          src.book_and_repo 'repo', 'Test', 'Some text.'
 
           # Allow the caller to customize the source
           before_first_build.call src, config
         end,
         before_second_build: before_second_build
       )
+      include_context 'build one book twice'
+    end
+
+    def self.build_one_book_out_of_one_repo_and_then_out_of_two(
```

Comment:
I'm assuming rubocop has already warned you about this method name length...?

Reply:
It didn't! It is certainly really long though....